packages/editor/src/components/editor/index.js

```diff
@@ -58,6 +58,10 @@ function Editor( {
 		[ postType, postId, templateId ]
 	);
 
+	if ( ! post ) {
```

Comment:
When this happens, is it an error, or is the post loading?

Reply:
The post is loading